SdlLogs/SdlLogs.py

```diff
@@ -131,7 +133,7 @@ def hide_show_region(self, pattern):
         self.view.unfold(trc_regions)
 
 
-def open_file(self, path_to_file, message_if_absent):
+def open_file_with_check_exist(self, path_to_file, message_if_absent):
```

Comment:
@MaximRybak I would say that old name was better.

Reply:
Fixed in c82425a84f05a787910dbe0681d7d4b1f6a15a91